docs/sources/release-notes/v2.4.md

```diff
@@ -0,0 +1,89 @@
+
+## Features and enhancements
+
+- **Query-scheduler ring-based service discovery**:
+  The [query-scheduler]({{< relref "../operators-guide/architecture/components/query-scheduler/index.md" >}}) is an optional, stateless component that retains a queue of queries to execute, and distributes the workload to available queriers. The use the query-scheduler, query-frontends and queriers are required to discover the addresses of the query-scheduler instances.
+
+  Other than the DNS-based service discovery, Mimir 2.4 introduces the ring-based service discovery for the query-scheduler. When enabled, the query-schedulers join their own hash ring (similar to other Mimir components), and the query-frontends and queriers discover query-scheduler instances via the ring.
+
+  The ring-based service discovery makes easier to setup the query-scheduler in environments where you can't easily define a DNS entry resolving to the running query-scheduler instances. Refer to [query-scheduler configuration]({{< relref "../operators-guide/architecture/components/query-scheduler/index.md#configuration" >}}) for more information.
+
+- **New API endpoint exposing per-tenant limits**:
+  Mimir 2.4 introduces a new API endpoint, available on all Mimir components that load the runtime configuration, exposing the limits for the authenticated tenant. This new API endpoint can be used when developing custom integrations with Mimir that require looking up the actual limits applied on a given tenant. Refer to [Get tenant limits]({{< relref "../operators-guide/reference-http-api/index.md#get-tenant-limits" >}}) for more information.
+
+- **New TLS configuration options**:
+  Mimir 2.4 introduces new options to configure the accepted TLS cipher suites and minimum version for the HTTP and gRPC clients used between Mimir components, or by Mimir to communicate to external services (e.g. Consul or etcd).
+
+  These new configuration options can be used to override the default TLS settings and meet your security policies. Refer to [Securing Grafana Mimir communications with TLS]({{< relref "../operators-guide/secure/securing-communications-with-tls.md" >}}) for more information.
+
+- **Maximum range query length limit**:
+  Mimir 2.4 introduces the new configuration option `-query-frontend.max-total-query-length` to limit the maximum range query length, computed as the query's `end` minus `start` timestamp. This limit is enforced in the query-frontend and defaults to `-store.max-query-length` if unset.
+
+  The new configuration option allows you to set different limits between the received query maximum length (`-query-frontend.max-total-query-length`) and the maximum length of partial queries after splitting and sharding (`-store.max-query-length`).
+
+### Helm chart improvements
+
+The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of the Mimir 2.4 release, we’re also releasing version 3.2 of the Mimir Helm chart.
```

Comment:
```suggestion
The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes. As part of the Mimir 2.4 release, we’re also releasing version 3.2 of the `mimir-distributed` Helm chart.
```

not sure what we've used in other release notes, but that's how we refer to it in docs. It's less ambiguous which chart this means.